src/bootstrap/doc.rs

```diff
@@ -527,7 +527,10 @@ impl Step for Rustc {
         cargo.rustdocflag("--document-private-items");
         cargo.rustdocflag("--enable-index-page");
         cargo.rustdocflag("-Zunstable-options");
-        cargo.rustdocflag("-Znormalize-docs");
+        // cfg(not(bootstrap)), can be removed on the next beta bump
+        if stage != 0 {
```

Comment:
Nit, but might it be clearer as

```suggestion
        if stage > 0 {
```

?

Reply:
`stage` seems to be a `u32`, so it should never be negative. (I wonder why it's not a `u8`...)

Reply:
I think this is less clear, actually - it obscures that the only time this is true is when the stage is 0.

> (I wonder why it's not a u8...)

Probably no one thought of it; performance doesn't really matter for rustbuild.